app/models/item.rb

```diff
@@ -160,25 +168,24 @@ def language
 
   private
 
-    def valid_http_uri?(uri)
-      URI.parse(uri).kind_of?(URI::HTTP)
-    rescue URI::InvalidURIError
-      false
-    end
-
-    def transform_dotted_keys(doc)
-      doc.keys
-        .select { |k| k.index('.') }
-        .select { |k| k =~ /^(sourceResource|object)/ }
-        .each do |k|
-          value = doc.delete k
-          tokens = k.split '.'
-          first_token = tokens.shift
-          tokens.reverse.each { |t| value = {t => value} }
-          doc[first_token] ||= {}
-          doc[first_token].deep_merge! value
-        end
-      doc
-    end
-
+  def valid_http_uri?(uri)
+    URI.parse(uri).kind_of?(URI::HTTP)
+  rescue URI::InvalidURIError
+    false
+  end
+
+  def transform_dotted_keys(doc)
+    doc.keys
+      .select { |k| k.index('.') }
+      .select { |k| k =~ /^(sourceResource|object)/ }
+      .each do |k|
+        value = doc.delete k
+        tokens = k.split '.'
+        first_token = tokens.shift
+        tokens.reverse.each { |t| value = {t => value} }
+        doc[first_token] ||= {}
+        doc[first_token].deep_merge! value
+      end
+    doc
+  end
```

Comment:
No changes were made to the above two methods aside from fixing the indentation to align with our team's style guide.